src/igvfd/schemas/biosample.json

```diff
@@ -23,9 +17,6 @@
         {
             "$ref": "mixins.json#/schema_version"
         },
-        {
```

Comment:
I believe I removed it because uuid is unused in the abstract level, it doesn't seem to have any consequence in removing it as I removed it from sample as well.